R/pkg/R/column.R

```diff
@@ -169,7 +169,7 @@ setMethod("alias",
 #' @note substr since 1.4.0
 setMethod("substr", signature(x = "Column"),
           function(x, start, stop) {
-            jc <- callJMethod(x@jc, "substr", as.integer(start - 1), as.integer(stop - start + 1))
+            jc <- callJMethod(x@jc, "substr", as.integer(start), as.integer(stop - start + 1))
```

Comment:
I think we should do two things:
1. add to the func doc that the `start` param should be 0-base and to add to the example with the result
`collect(select(df, substr(df$a, 0, 5))) # this should give you...`

Reply:
I think you mean 1-based,

Reply:
Added to the func doc.